components/bl-AnimateOnScroll-component/src/index.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useMemo, useRef } from 'react';
+import { Aos } from './aos';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const DevicesMap = {
+  mobile: 'mobile',
+  tablet: 'tablet',
+  all   : true,
+  none  : false,
+};
+
+export default function AnimateOnScroll({ component, eventHandlers, pods }) {
```

Comment:
have you tried to use more than one instance on the same page with different settings?

Reply:
yes, all work well